src/views/Material.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <v-app-bar
+      :clipped-left="$vuetify.breakpoint.lgAndUp"
+      app
+      color="blue-grey darken-3"
+      dark
+    >
+      <v-avatar :tile="true" width="140px">
```

Comment:
Could factor this out as appbar-shared-left or similar. Same for Order component.